### PR TITLE
Throttle correction comment

### PR DESCRIPTION
**Pull request by avoid3d**
Review this request first: #405, then only compare new changes onto that one, or ask me to rebase :)

Added clarifying comment to the throttle correction function in imu.h.

### CODE REVIEW

**Comment by hydra** [comment]
this all looks good, will merge

File: src/main/common/maths.h [hunk 1]
```diff
@@ -21,16 +21,10 @@
...
-#endif
-#define M_PI       3.14159265358979323846f
-
-#define RADX10 (M_PI / 1800.0f)                  // 0.001745329252f
-#define RAD    (M_PI / 180.0f)
+// Use floating point M_PI instead explicitly.
+#define M_PIf       3.14159265358979323846f
 
-#define DEG2RAD(degrees) (degrees * RAD)
+#define RAD    (M_PIf / 180.0f)
```
Comment (thenickdude) [inline src/main/common/maths.h]: Why use floating point versions of these constants? Wouldn't the result be ever-so-slightly more accurate if it was (float) (M_PI / 180.0), with still no double support required at runtime?

Comment (hydra) [inline src/main/common/maths.h]: I guess my question would be: Is it better to have a consistant definition of Pi or greater accuracy.

Perhaps we should define M_PId and use appropriate casting as you suggest in the macros to make it clearer?

Comment (ledvinap) [inline src/main/common/maths.h]: My gcc produces identical result ...

Comment (thenickdude) [inline src/main/common/maths.h]: ledvinap: True dat.

Comment (hydra) [inline src/main/common/maths.h]: so what should we do here then guys? leave as is?

Comment (thenickdude) [inline src/main/common/maths.h]: Personally I'd prefer the cast, but testing proves that it doesn't make a
difference what way it's done! (so there's no reason to tweak it)

**Pull request merged**